pkg/kubelet/apis/config/types.go

```diff
@@ -506,7 +506,7 @@ type KubeletConfiguration struct {
 	// container restart behavior
 	// +featureGate=KubeletCrashLoopBackoffMax
 	// +optional
```

Comment:
What does optional mean if we're defaulting one of the sub-fields?

Reply:
optional means that clients do not have to specify it.  These tags are from the POV of the client.  It's not 100% consistent, but that is the intention.

Reply:
I left this here since this statement seems to be true to me -- clients don't have to specify since it is only accepted if the feature gate is also on. Holler at me if this is a dealbreaker.